### PR TITLE
Addition Methods: New method nisBr (#2196)

### LINKED ISSUE #2169: Confirm Password must be matched with Password
### Your environment
* version of `jquery-validate`:
* which browser and its version:

### Expected behaviour
it should check it match

### Actual behaviour
it actually says it dont match but it is

### PR DESCRIPTION
**Pull request by csmendonca**
Fixes new implementation
Fixes #2196 

#### Description

Thank you!

### CODE REVIEW

File: src/additional/nisBR.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ * Brazilian tax registration number NIS of PIS numbers have 11 digits in total: 10 numbers followed by 1 check numbers
+ * that are being used for validation.
+ * @copyright (c) 25/10/2016 22:40, Cleiton da Silva Mendonça
+ * @author Cleiton da Silva Mendonça <cleiton.mendonca@gmail.com>
+ * @link http://gitlab.com/csmendonca Gitlab of Cleiton da Silva Mendonça
+ * @link http://github.com/csmendonca Github of Cleiton da Silva Mendonça
+ */
+
+$.validator.addMethod( "nisBR", function( value ) {
+
```
Comment (Arkni) [inline src/additional/nisBR.js]: Please remove this empty line.

File: src/additional/nisBR.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/**
+ * Created for project jquery-validation.
+ * @Description Brazillian PIS or NIS number (Número de Identificação Social Pis ou Pasep) is the equivalent of a
+ * Brazilian tax registration number NIS of PIS numbers have 11 digits in total: 10 numbers followed by 1 check numbers
+ * that are being used for validation.
+ * @copyright (c) 25/10/2016 22:40, Cleiton da Silva Mendonça
```
Comment (Arkni) [inline src/additional/nisBR.js]: I think this is a copy-paste error. Can you please adjust the date to the right one.

File: src/additional/nisBR.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+	// Checking value to have 11 digits only
+	if ( value.length !== 11 ) {
+		return false;
+	}
+
+	//Get check number of value
+	cn = parseInt( value.substring( 10, 11 ), 10 );
+
+	//Get number with 10 digits of the value
```
Comment (Arkni) [inline src/additional/nisBR.js]: Please add a space before this comment and the one above it in order to be consistent with the rest of the comments.

**Comment by csmendonca** [comment]
Hi @Arkini, 

Thanks for your attention.
I'm going to do that changes.

**Pull request closed without merging**